docs/website/index.md

```diff
@@ -4,18 +4,13 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
 # Welcome to the documentation of Ikarus
-Ikarus is a finite element software originated at the [Institute for Structural Mechanics](https://www.ibb.uni-stuttgart.de/en/) at the university of Stuttgart.
+Ikarus is a finite element software program that originated at the [Institute for Structural Mechanics](https://www.ibb.uni-stuttgart.de/en/) at the University of Stuttgart.

```

Comment:
Since ikarus is not a single program but more a library or framework, we should write here like "Ikarus is a finite element framework ...".
and additionally something like "Ikarus provides the tools to create your own examples and rapid prototype finite element solutions schemes. This is done by using template meta programming in C++ to write generic code, which is the compiled for your example. This also provides more efficiency in constrast to a one fits all program executable." What do you think?